src/splat.ts

```diff
@@ -139,6 +139,7 @@ class Splat extends Element {
     worldBoundStorage: BoundingBox;
     localBoundDirty = true;
     worldBoundDirty = true;
+    visible_ = true;
```

Comment:
What's with the trailing underscore? Don't we normally prepend it for private class fields?

Reply:
yes I guess. will fix in a followup PR. thanks!